sublime/pythonLiveCoding.py

```diff
@@ -0,0 +1,306 @@
+import os
```

Comment:
One more thing: could you please change the file name to `python_live_coding.py`? You'll probably need to change the package folder name and settings file to match.

Reply:
Can do. I think sublime plugin's convention is upper-leading camel case however - I'm not sure if there's a technical requirement for this to do with name mangling / mapping but that's how I've seen sublime plugins done in the past. Happy to make the change though.

Reply:
I just tried it here, and `python_live_coding.py` works fine.

Reply:
Ok, I've implemented an on_idle callback. It's not the prettiest thing in the world, but it does the trick.